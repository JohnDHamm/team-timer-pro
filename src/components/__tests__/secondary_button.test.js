import React from 'react'
import renderer from 'react-test-renderer'
import SecondaryButton from '../secondary_button'

describe('<SecondaryButton />', () => {
  const label = 'save';
  const color = 'green';

  test('it renders correctly', () => {
    const tree = renderer.create(<SecondaryButton label={label} color={color} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
