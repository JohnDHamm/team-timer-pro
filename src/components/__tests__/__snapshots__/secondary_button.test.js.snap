// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SecondaryButton /> it renders correctly 1`] = `
<View
  style={
    Object {
      "borderColor": "green",
      "borderRadius": 3,
      "borderWidth": 1,
      "paddingHorizontal": 15,
      "paddingVertical": 5,
    }
  }
>
  <Text
    style={
      Object {
        "color": "green",
        "fontFamily": "dosis-regular",
        "fontSize": 20,
      }
    }
  >
    save
  </Text>
</View>
`;
